server/models/index.js

```diff
@@ -82,6 +82,8 @@ export function setupModels() {
   /**
    * Relationships
    */
+  // Activity
+  m.Activity.hasMany(m.Collective, { foreignKey: 'ParentCollectiveId', as: 'collective' });
```

Comment:
This doesn't look necessary, but even if it is the `as` is wrong: you want `activity.getCollective` to return the collective, not the parent.

Reply:
Ops, I was experimenting with this one and forgot to remove it. So yes this isn't necessary. 👍🏽